### PR TITLE
TensoRF go brrrrrrr

### PR DESCRIPTION
**Pull request by mcallisterdavid**
Sampling bug fixed. ~28k rays/sec -> ~170k rays/sec. Seems like expected performance level--faster than nerfacto for me. Don't understand why tests are failing if y'all have any ideas.

tensorf-speed-comparison
New TensoRF (left) vs. Old TensoRf (right)

### CODE REVIEW

**Comment by mcallisterdavid** [comment]
✨

File: nerfstudio/models/tensorf.py [hunk 1]
```diff
@@ -70,8 +70,10 @@ class TensoRFModelConfig(ModelConfig):
     """specifies a list of iteration step numbers to perform upsampling"""
     loss_coefficients: Dict[str, float] = to_immutable_dict({"rgb_loss": 1.0})
     """Loss specific weights."""
-    num_samples: int = 256
+    num_samples: int = 50
     """Number of samples in field evaluation"""
+    num_uniform_samples: int = 200
+    """Number of samples in density evalutaion"""
```
Comment (f-dy) [inline nerfstudio/models/tensorf.py:76]: typo

Comment (mcallisterdavid) [reply nerfstudio/models/tensorf.py:76]: Fixed, thanks!

**Review by tancik — APPROVED** [review]
LGTM, the test errors were due to something else that should be fixed now.

**Pull request merged**